packages/connectivity_plus/example/pubspec.yaml

```diff
@@ -0,0 +1,30 @@
+name: connectivity_plus_example
+description: Demonstrates how to use the connectivity_plus_tizen plugin.
+publish_to: "none"
+
+dependencies:
+  connectivity_plus: ^1.0.1
+  connectivity_plus_tizen:
+    path: ../
+  flutter:
+    sdk: flutter
+
+dev_dependencies:
+  flutter_driver:
+    sdk: flutter
+  flutter_test:
+    sdk: flutter
+  integration_test:
+    sdk: flutter
+  integration_test_tizen:
+    path: ../../integration_test/
+
+dependency_overrides:
+  args: ^2.0.0
```

Comment:
I don't think this is needed anymore
```suggestion
```

Reply:
Recent migration to Flutter 2.2.1(args 1.6.0 -> args 2.1.0) by flutter-tizen/flutter-tizen/pull/138 solved this. Thanks for noticing!